### DISCUSSION TITLE
Sturgeon's Law is a saying that 90% of everything is ...

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1149] [Galaxy Trivia] [answered]
**Bonus:** Where in the house might you have a ramekin?

**Comment by tomerf** [comment]
Sturgeon's law is the adage that “ninety percent of everything is **crap**”. This suggests that, in general, the vast majority of the works that are produced in any given field are likely to be of low quality.

A ramekin is a small dish used for culinary purposes and thus would be found in the kitchen.

**Comment by thuongtruong1009** [comment, marked as answer]
Sturgeon's Law is a saying coined by science fiction writer Theodore Sturgeon. It states that "90% of everything is crap." In essence, it suggests that the vast majority of any given category, whether it's literature, art, technology, or any other field, may be of low quality or unremarkable, while only a small percentage is of high quality or truly valuable. This principle is often used to emphasize the importance of sifting through a large quantity of content or creations to find the gems or worthwhile contributions.

A ramekin is typically found in the kitchen, specifically in the cooking and baking area. It's a small, shallow, and usually ceramic or glass dish that is often used for individual servings of baked dishes, such as crème brûlée, soufflé, or pot pies. Ramekins can also be used for serving condiments, sauces, or small snacks. So, you would commonly find ramekins in the kitchen, stored with other cookware and bakeware.
